V1/auth/validate.py

```diff
@@ -0,0 +1,58 @@
+from flask import make_response, jsonify
+import re
+
+
+def response(status, message, status_code):
+    """
+    Helper method to make an Http response
+    :param status: Status
+    :param message: Message
+    :param status_code: Http status code
+    :return:
+    """
+    return make_response(jsonify({
+        'status': status,
+        'message': message
+    })), status_code
+
+
+def empty(**data):
+    '''method to validate username input'''
+    messages = {}
+    for key in data:
+        newname = re.sub(r'\s+', '', data[key])
+        if not newname:
+            message = {'message': key + ' cannot be an empty string'}
+            messages.update({key + '-Error:': message})
+    return messages
+
+
+def whitespace(data):
+    '''method to validate white'''
+    newname = re.sub(r'\s+', '', data)
+    afterlength = len(newname)
+    actuallength = len(data)
+    if afterlength != actuallength:
+        return True
+
+
+def val_none(**data):
+    '''method to check none'''
```

Comment:
A good example of a comment within a function would be """Takes in ausernameand returns aboolean """